composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "faktiva/prestashop-seo-tk",
+    "name": "faktiva/faktiva_seo_tk",
```

Comment:
Hi @drAlberT 

The modification of the name is because the folder name of the module, php file and php class must be align.

Composer use the second part of the name faktiva/**prestashop-seo-tk** as the folder of the installation.

I decided to change it, because **prestashop-seo-tk** is not compatible with php classes (the hyphens).

As you mention using the field extra, I think it is a better alternative because we don't change the package name.

```
{
    "name": "aktiva/prestashop-seo-tk",
    "extra": {
        "installer-name": "faktiva_seo_tk"
    }
}
```

another option would be convert it to camelCase the class name *faktivaSeoTk* 

http://doc.prestashop.com/display/PS16/Creating+a+first+module